src/activities/content/ContentFileEntity.js

```diff
@@ -184,7 +184,8 @@ export class ContentFileEntity extends ContentWorkingCopyEntity {
 	equals(contentFile) {
 		const diffs = [
 			[this.title(), contentFile.title],
-			[this.getFileHref(), contentFile.fileHref]
+			[this.getFileHref(), contentFile.fileHref],
+			...(contentFile.fileType !== FILE_TYPES.html ? [[this.descriptionRichText(), contentFile.descriptionRichText]] : [])
```

Comment:
nit: is checking for the html file type necessary?

Reply:
yes, `ContentFileEntity` is a parent class for all the uploaded content files and the equals method is common for all of them. Among those, for HTML file type, we do not have the `descriptionRichText` field. Hence, put a condition to check for that field only in case of non html file types.